guides/decoding-blocks-transactions/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "decoding-blocks-transactions",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Would this be a problem?

Reply:
I don't think so. It is the default license when doing `npm init -y`